src/doctestfilters.jl

```diff
@@ -1,11 +1,19 @@
 # Define some replacements which are applied in `docs/make.jl` and `test/testmanual.jl`.
+# The idea is that the `jldoctest`ed examples in the source files
+# have the format that fits to the "current" Julia,
+# and that newer Julia versions (Julia nightly) may show a different format.
+# Each replacement in `GAP_doctestfilters` gets applied to both the
```

Comment:
A regex in itself is not a replacement, so maybe clarify that each regex in `GAP_doctestfilters` defines implicitly a replacement: text matching any of the regexes is replaced by the empty string.

Reply:
Thanks. Yes, it is better to say what actually happens.